lib/builderator/control/version.rb

```diff
@@ -36,9 +36,7 @@ def current
 
         def set_config_version
           Config.defaults.version = current.to_s
-
-          Config.recompile ## Update compiled version attribute
-          Config.recompile ## Pass new attribute back out to other layers
+          Config.recompile
```

Comment:
It looks like you changed all the recompiles to compile except this one. Was it meant to be left?